src/db/effect_bundles_tables_vco.tsv

```diff
@@ -7,11 +7,12 @@ vco_dilemma_payload_vmp_man_ossified_portal_third	I Care Not	Sylvania is present
 vco_victory_payload_brt_blessed_by_the_lady	Blessed by The Lady	Alberic has supped from the grail, becoming a god amongst men.	vow_grail_positive.png	faction	1	true	false	true
 vco_victory_payload_brt_bordeleaux_reclaimed	Bordeleaux Reclaimed	The peasants are safe. Have them return to their labours, for idle hands are the daemon's playthings.	peasant.png	faction	1	true	false	true
 vco_victory_payload_brt_the_sea_gods_wrath	The Sea God's Wrath	The scurge of undeath has been banished from the waves. Manann is pleased.	pirate_crew.png	faction	1	true	false	true
-vco_victory_payload_cst_sar_1_captain_of_the_swordfysh	The Pirate Queen	Do you know what she, the queen of the pirates, did when the treasure she always looked for was placed in front of her very eyes? She laughed.	pirate_cove.png	faction	1	true	false	true
-vco_victory_payload_cst_sar_2_daughter_of_manann_1	Lady of the Seas	Manann is said to be mighty, called the Summoner of Storms, and Master of Maelstroms.... But no-one ever said he pays the crews wages. Sacking cities does that.	pirate_crew.png	faction	1	true	false	true
-vco_victory_payload_cst_sar_2_daughter_of_manann_2	Manann's Favoured	Many a victim has died at sea, their corpses sinking gracefully to the sea bed. Manann is all too happy to raise those corpses as fodder for those he favours.	cst_sea_monster.png	faction	1	true	false	true
-vco_victory_payload_cst_sar_3_dead_man_s_vengeance_1	Sank the Bloody Reaver	Minds filled with vengeance, eyes filled with hatred, hearts filled with satisfaction... The sacking of Sartosa has been repaid tenfold, and the undead of Count Noctilus, once Nyklaus Von Carstein, lie still again.	ranged_damage.png	faction	1	true	false	true
-vco_victory_payload_cst_sar_3_dead_man_s_vengeance_2	Master of the Colossi	The seas are a better place without the Dreadfleet and it's ravaging captain, but his mastery of the Necrofex Colossus was impressive. Such knowledge should not sink beneath the waves.	edict_woc_portent_of_doom.png	faction	1	true	false	true
+vco_victory_payload_cst_sar_1_captain_of_the_swordfysh	Jaego Roth's Moondail	This moondail once belonged to Captain Jaego Roth, before he was slain by Count Noctilus. Any pirate worth their name pays their respects on seeing it... Mainly in additional coin to it's new bearer.	wh2_dlc11_item_abilities_captain_roths_moondial.png	faction	1	true	false	true
+vco_victory_payload_cst_sar_1_pirate_queen	The Pirate Queen	From Erengrad to Fu-Chow, all fear the name: Aranessa Saltspite!	bullet_cst_infamy.png	faction	1	true	false	true
```

Comment:
Bruh @OhManTFE why do you remove the flavour? The `She laughed` thing is my fav :(